packages/core/src/external/cda/process-attachments.ts

```diff
@@ -64,13 +73,11 @@ export async function processAttachments({
   const docRefs: DocumentReference[] = [];
   const uploadDetails: UploadParams[] = [];
 
-  b64Attachments.map(async att => {
-    const act = att.act as ConcernActEntryAct;
+  b64Attachments.acts.map(act => {
+    const fileDetails = getDetailsForAct(act.text);
```

Comment:
`getDetailsForActText` or pass in act and dot-notation into text within the function.